terraform/damages.tf

```
@@ -0,0 +1,19 @@
+locals {
+  dm_non_prod_domains = ["mail-damages-nonprod.platform.hmcts.net"]
+  dm_prod_domains     = ["mail-damages.platform.hmcts.net"]
```

dm means document management, definitely don't use this abbreviation.
```suggestion
  damages_prod_domains     = ["mail-damages.platform.hmcts.net"]
```

---

done